fix ryuring-17730  /lib/Baser/Config/Theme/　多言語化

katokaisya opened the pull request:
/lib/Baser/Config/Theme/　の多言語化

Commit:
fix ryuring-17730  /lib/Baser/Config/Theme/　多言語化

Commit:
fix ryuring-17730 /lib/Baser/Config/Theme/　多言語化　submitの内容がPOTファイルに準拠していなかったため、修正

ryuring commented on lib/Baser/Config/theme/bc_sample/Contents/search.php:
```diff
@@ -10,7 +10,7 @@
 <?php if (!empty($this->Paginator)): ?>
 <div class="search-result">
-	<?php echo $this->Paginator->counter(array('format' => '<strong>' . implode(' ', $query) . '</strong> で検索した結果 <strong>%start%〜%end%</strong>件目 / %count% 件')) ?>
+	<?php echo $this->Paginator->counter(array('format' => '<strong>' . implode(' ', $query) . '</strong>'.__('baser',' で検索した結果').' <strong>%start%〜%end%</strong>'.__('baser','件目').' / %count% '.__('baser','件'))) ?>
```
ここはフロントに出る情報なので、__() でお願いします。

katokaisya replied:
9026c70647b72e0e4f694b79bac8587912663f9b
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Contents/search.php:
```diff
@@ -27,7 +27,7 @@
 <?php else: ?>
 <div>
-	<p class="no-data">該当する結果が存在しませんでした。</p>
+	<p class="no-data"><?php echo __('baser', '該当する結果が存在しませんでした。'); ?></p>
```
__() でお願いします。

katokaisya replied:
9026c70647b72e0e4f694b79bac8587912663f9b
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Contents/smartphone/search.php:
```diff
@@ -11,7 +11,7 @@
 <?php if (!empty($this->Paginator)): ?>
 	<div class="search-result">
-		<?php echo $this->Paginator->counter(array('format' => '<strong>' . implode(' ', $query) . '</strong> で検索した結果 <strong>%start%〜%end%</strong>件目 / %count% 件')) ?>
+		<?php echo $this->Paginator->counter(array('format' => '<strong>' . implode(' ', $query) . '</strong>'.__(' で検索した結果').' <strong>%start%〜%end%</strong>'.__('件目').' / %count% '.__('件'))) ?>
```
半角スペースは関数の外に出してください。

katokaisya replied:
9026c70647b72e0e4f694b79bac8587912663f9b
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Elements/blog_tag.php:
```diff
@@ -8,6 +8,6 @@
 <?php if (!empty($this->Blog->blogContent['tag_use'])): ?>
 	<?php if (!empty($post['BlogTag'])) : ?>
-		<div class="tag">タグ：<?php $this->Blog->tag($post) ?></div>
+		<div class="tag"><?php echo __(タグ) ?>：<?php $this->Blog->tag($post) ?></div>
```
シングルコーテーションが入ってません。

katokaisya replied:
9026c70647b72e0e4f694b79bac8587912663f9b
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Elements/paginations/simple.php:
```diff
@@ -17,8 +17,8 @@
 <?php if ((int) $this->Paginator->counter(array('format' => '%pages%')) > 1): ?>
 <div class="pagination">
-	<?php echo $this->Paginator->prev('< 前へ', array('class' => 'prev'), null, array('class' => 'disabled')) ?>
+	<?php echo $this->Paginator->prev(__('< 前へ'), array('class' => 'prev'), null, array('class' => 'disabled')) ?>
```
記号は外に出してください。

katokaisya replied:
9026c70647b72e0e4f694b79bac8587912663f9b
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Elements/paginations/simple.php:
```diff
@@ -17,8 +17,8 @@
+	<?php echo $this->Paginator->prev(__('< 前へ'), array('class' => 'prev'), null, array('class' => 'disabled')) ?>
 	<?php echo $this->Html->tag('span', $this->Paginator->numbers(array('separator' => '', 'class' => 'number', 'modulus' => $modules), array('class' => 'page-numbers'))) ?>
-	<?php echo $this->Paginator->next('次へ >', array('class' => 'next'), null, array('class' => 'disabled')) ?>
+	<?php echo $this->Paginator->next(__('次へ >'), array('class' => 'next'), null, array('class' => 'disabled')) ?>
```
記号は外に出してください。

katokaisya replied:
9026c70647b72e0e4f694b79bac8587912663f9b
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Elements/smartphone/blog_tag.php:
```diff
@@ -8,7 +8,7 @@
 <?php if (!empty($this->Blog->blogContent['tag_use'])): ?>
 	<?php if (!empty($post['BlogTag'])) : ?>
-		<div class="tag">タグ：<?php $this->Blog->tag($post) ?></div>
+		<div class="tag"><?php echo __(タグ) ?>：<?php $this->Blog->tag($post) ?></div>
```
コーテーションがありません。

katokaisya replied:
9026c70647b72e0e4f694b79bac8587912663f9b
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Elements/smartphone/list_num.php:
```diff
@@ -31,7 +31,7 @@
 <?php if ($link): ?>
 	<div class="list-num">
-		<strong>表示件数</strong><p><?php echo $link ?></p>
+		<strong>__('表示件数')</strong><p><?php echo $link ?></p>
```
PHPタグが抜けてます。

katokaisya replied:
9026c70647b72e0e4f694b79bac8587912663f9b
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Elements/smartphone/paginations/simple.php:
```diff
@@ -18,9 +18,9 @@
 <?php if ((int) $this->Paginator->counter(array('format' => '%pages%')) > 1): ?>
 	<div class="pagination clearfix">
-		<?php echo $this->Paginator->prev('< 前へ', array('class' => 'prev'), null, array('class' => 'disabled')) ?>
+		<?php echo $this->Paginator->prev(__('< 前へ'), array('class' => 'prev'), null, array('class' => 'disabled')) ?>
```
記号外へ

ryuring commented on lib/Baser/Config/theme/bc_sample/Elements/smartphone/paginations/simple.php:
```diff
@@ -18,9 +18,9 @@
+		<?php echo $this->Paginator->prev(__('< 前へ'), array('class' => 'prev'), null, array('class' => 'disabled')) ?>
 		<?php echo $this->Html->tag('span', $this->Paginator->numbers(array('separator' => '', 'class' => 'number', 'modulus' => $modules), array('class' => 'page-numbers'))) ?>
-		<?php echo $this->Paginator->next('次へ >', array('class' => 'next'), null, array('class' => 'disabled')) ?>
+		<?php echo $this->Paginator->next(__('次へ >'), array('class' => 'next'), null, array('class' => 'disabled')) ?>
```
記号外へ

katokaisya replied:
9026c70647b72e0e4f694b79bac8587912663f9b
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Elements/smartphone/widgets/blog_calendar.php:
```diff
@@ -101,27 +101,27 @@
-<th>木</th>
-<th>金</th>
-<th class="saturday">土</th>
+<th class="sunday">' . __('日') . '</th>
```
PHPタグが抜けてます。

ryuring replied:
以下、6行もお願いします。

katokaisya replied:
print 'string '. __('日本語'). 'string';
という形なのですが、<?php echo  __('日本語') ?>は必要ですか？

ryuring commented on lib/Baser/Config/theme/bc_sample/Elements/widgets/blog_calendar.php:
```diff
@@ -101,27 +101,27 @@
-<th>木</th>
-<th>金</th>
-<th class="saturday">土</th>
+<th class="sunday">' . __('日') . '</th>
```
PHPタグ抜け、以下６行も

katokaisya replied:
print 'string '. __('日本語'). 'string';
という形なのですが、<?php echo  __('日本語') ?>は必要ですか？

Commit:
fix ryuring-17730 言語関数置き換え処理

lib/Baser/Config/setting.php
lib/Baser/Config/update/4.0.1/config.php
lib/Baser/Config/update/4.0.1/updater.php

ryuring commented on lib/Baser/Config/theme/bc_sample/Elements/widgets/blog_monthly_archives.php:
```diff
@@ -48,9 +48,9 @@
 				<?php endif ?>
 				<?php if ($view_count): ?>
-					<?php $title = $postedDate['year'] . '年' . $postedDate['month'] . '月' . '(' . $postedDate['count'] . ')' ?>
+					<?php $title = $postedDate['year'] . '/' . $postedDate['month'] . '(' . $postedDate['count'] . ')' ?>
```
lib/Baser/Config/theme/bc_sample/Elements/smartphone/widgets/blog_monthly_archives.php と対応が違います。
どうしましょうか。

ryuring commented on lib/Baser/Config/theme/bc_sample/Elements/widgets/blog_monthly_archives.php:
```diff
@@ -48,9 +48,9 @@
+					<?php $title = $postedDate['year'] . '/' . $postedDate['month'] . '(' . $postedDate['count'] . ')' ?>
 				<?php else: ?>
-					<?php $title = $postedDate['year'] . '年' . $postedDate['month'] . '月' ?>
+					<?php $title = $postedDate['year'] . '/' . $postedDate['month'] ?>
```
lib/Baser/Config/theme/bc_sample/Elements/smartphone/widgets/blog_monthly_archives.php と対応が違います。
どうしましょうか。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/blog_comment_admin.php:
```diff
@@ -8,26 +8,26 @@
 
-　<?php echo $Content['title'] ?> へのコメントを受け付けました。
-　受信内容は下記のとおりです。
+　<?php echo $Content['title'] ?> <?php echo __('へのコメントを受け付けました。
```
文字列の途中でそのまま改行を入れてはいけません。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/blog_comment_admin.php:
```diff
@@ -8,26 +8,26 @@
 <?php echo ($BlogComment['message']) ?>　
 
-コメントの公開状態を変更する場合は次のURLよりご確認ください。
+<?php echo __('コメントの公開状態を変更する場合は次のURLよりご確認ください。') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/blog_comment_admin.php:
```diff
@@ -8,26 +8,26 @@
-　<?php echo $Content['title'] ?> へのコメントを受け付けました。
-　受信内容は下記のとおりです。
+　<?php echo $Content['title'] ?> <?php echo __('へのコメントを受け付けました。
+　受信内容は下記のとおりです。') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/mail_default.php:
```diff
@@ -7,19 +7,19 @@
 <?php if ($other['mode'] === 'user'): ?>
-　この度は、お問い合わせいただきありがとうございます。 
-　送信内容は下記のようになっております。  
+  <?php echo __('この度は、お問い合わせいただきありがとうございます。') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/mail_default.php:
```diff
@@ -7,19 +7,19 @@
 <?php elseif ($other['mode'] === 'admin'): ?>
-　<?php echo $mailConfig['site_name'] ?> へのお問い合わせを受け付けました。
-　受信内容は下記のとおりです。   
+　<?php echo $mailConfig['site_name'] ?> <?php echo __('へのお問い合わせを受け付けました。') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/mail_default.php:
```diff
@@ -28,12 +28,12 @@
-　なお、このメールは自動返信メールとなっております。 
-　メールを確認させて頂き次第、早急にご連絡させていただきます。 
-　恐れ入りますがしばらくお待ちください。 
+  <?php echo __('なお、このメールは自動返信メールとなっております。 
```
文字列の途中で改行を入れないようにしてください。個別にタグを書いてください。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/mail_default.php:
```diff
@@ -28,12 +28,12 @@
 <?php elseif ($other['mode'] === 'admin'): ?>
-　なお、このメールは自動転送システムです。 
-　受け付けた旨のメールもユーザーへ送られています。 
+  <?php echo __('なお、このメールは自動転送システムです。 
```
文字列の途中で改行を入れないようにしてください。個別にタグを書いてください。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/mail_default.php:
```diff
@@ -28,12 +28,12 @@
-　なお、このメールは自動転送システムです。 
-　受け付けた旨のメールもユーザーへ送られています。 
+  <?php echo __('なお、このメールは自動転送システムです。 
+  　受け付けた旨のメールもユーザーへ送られています。') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/smartphone/mail_default.php:
```diff
@@ -7,19 +7,19 @@
 <?php echo $other['date']; ?>
 ━━━━━━━━━━━━━━
-　お問合せを受付けました
+　<?php echo __('お問合せを受付けました') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/smartphone/mail_default.php:
```diff
@@ -7,19 +7,19 @@
 <?php if ($other['mode'] === 'user'): ?>
-	　この度は、お問い合わせいただきありがとうございます。
-	　送信内容は下記のようになっております。
+	　<?php echo __('この度は、お問い合わせいただきありがとうございます。') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/smartphone/mail_default.php:
```diff
@@ -7,19 +7,19 @@
-	　この度は、お問い合わせいただきありがとうございます。
-	　送信内容は下記のようになっております。
+	　<?php echo __('この度は、お問い合わせいただきありがとうございます。') ?>
+	　<?php echo __('送信内容は下記のようになっております。') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/smartphone/mail_default.php:
```diff
@@ -7,19 +7,19 @@
 <?php elseif ($other['mode'] === 'admin'): ?>
-	　<?php echo $mailConfig['site_name'] ?> へのお問い合わせを受け付けました。
-	　受信内容は下記のとおりです。
+	　<?php echo $mailConfig['site_name'] ?> <?php echo __('へのお問い合わせを受け付けました。') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/smartphone/mail_default.php:
```diff
@@ -7,19 +7,19 @@
-	　<?php echo $mailConfig['site_name'] ?> へのお問い合わせを受け付けました。
-	　受信内容は下記のとおりです。
+	　<?php echo $mailConfig['site_name'] ?> <?php echo __('へのお問い合わせを受け付けました。') ?>
+	　<?php echo __('受信内容は下記のとおりです。') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/smartphone/mail_default.php:
```diff
@@ -7,19 +7,19 @@
 
 ━◇◆━━━━━━━━━━━
-　◆ お問い合わせ内容
+　◆ <?php echo __('お問い合わせ内容') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/smartphone/mail_default.php:
```diff
@@ -28,12 +28,12 @@
 
 <?php if ($other['mode'] === 'user'): ?>
-	　なお、このメールは自動返信メールとなっております。
+	　<?php echo __('なお、このメールは自動返信メールとなっております。
```
文字列の途中で改行を入れないようにしてください。個別にタグを書いてください。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/smartphone/mail_default.php:
```diff
@@ -28,12 +28,12 @@
 <?php elseif ($other['mode'] === 'admin'): ?>
-	　なお、このメールは自動転送システムです。
-	　受け付けた旨のメールもユーザーへ送られています。
+	　<?php echo __('なお、このメールは自動転送システムです。
```
文字列の途中で改行を入れないようにしてください。個別にタグを書いてください。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/smartphone/mail_default.php:
```diff
@@ -28,12 +28,12 @@
-	　なお、このメールは自動転送システムです。
-	　受け付けた旨のメールもユーザーへ送られています。
+	　<?php echo __('なお、このメールは自動転送システムです。
+	　受け付けた旨のメールもユーザーへ送られています。') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Emails/text/smartphone/mail_default.php:
```diff
@@ -28,12 +28,12 @@
+	　<?php echo __('なお、このメールは自動返信メールとなっております。
 	　メールを確認させて頂き次第、早急にご連絡させていただきます。
-	　恐れ入りますがしばらくお待ちください。
+	　恐れ入りますがしばらくお待ちください。') ?>
```
PHPタグの後、最後に半角、または、スペースをいれてください。改行が消されてしまいます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Mail/default/submit.php:
```diff
@@ -11,13 +11,12 @@
 <div>
-	<p>お問い合わせ頂きありがとうございました。<br>
-		確認次第、ご連絡させて頂きます。</p>
+	<p><?php echo __('お問い合わせ頂きありがとうございました。<br>確認次第、ご連絡させて頂きます。') ?></p>
```
<br>を内包しないでください。分けてください。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bc_sample/Mail/smartphone/default/submit.php:
```diff
@@ -11,13 +11,12 @@
 <div>
-	<p>お問い合わせ頂きありがとうございました。<br>
-		確認次第、ご連絡させて頂きます。</p>
+	<p><?php echo __('お問い合わせ頂きありがとうございました。<br>確認次第、ご連絡させて頂きます。') ?></p>
```
<br>を入れないでください。分けてください。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bccolumn/Blog/default/posts.php:
```diff
@@ -20,5 +20,5 @@
 	</ul>
 	<?php else: ?>
-	<p class="no-data">記事がありません</p>
+	<p class="no-data"><?php echo __('記事がありません') ?></p>
```
最後の。を入れるように統一されているようですが、ここは入っていません。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/bccolumn/Blog/topics/index.php:
```diff
@@ -40,7 +40,7 @@
 	<?php endforeach; ?>
 <?php else: ?>
-	<p class="no-data">記事がありません。</p>
+	<p class="no-data"><?php echo __('詳細ページへ') ?></p>
```
そもそも日本語が間違ってます。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring commented on lib/Baser/Config/theme/nada-icons/Mail/default/submit.php:
```diff
@@ -10,12 +10,11 @@
 <div class="section">
-	<p>お問い合わせ頂きありがとうございました。<br />
-		確認次第、ご連絡させて頂きます。</p>
+	<p><?php echo __('お問い合わせ頂きありがとうございました。<br>確認次第、ご連絡させて頂きます。') ?></p>
```
<br>を入れないでください。分けてお願いします。

katokaisya replied:
3f4b5cf60cd64cad626bc2733d9fa2c2f6088ddb
にて修正しました。

ryuring reviewed (commented):
@katokaisya レビュー完了しました。
指摘点が多すぎて辛いので、もう少しコミットの分割をお願いします。
また、指摘点について、今回のコードで他にないか、再チェックをお願いします。

Commit:
fix ryuring-17730 言語関数置き換え処理

lib/Baser/Config/setting.php
入力ミス　cake errorを出していたので、再コミット

Commit:
fix ryuring-17730 言語関数置き換え処理

 __('baser',　修正
半角スペースは関数の外に出す
シングルコーテーション漏れ対応
記号は外
PHPタグ漏れ修正

Commit:
fix ryuring-17730 言語関数置き換え処理

改行をなくす
メール本文のphpタグの末尾に半角スペース（改行用）
テキストの修正

Commit:
fix ryuring-17730 言語関数置き換え処理

カレンダーウィジェットの　当月が「2017/8」となっていたので、「2017/Aug」になるように調整
（OKなら、/plugin/Blog/Elements/widgets/にも適用します）
